src/vaadin-date-time-picker.html

```diff
@@ -179,13 +178,17 @@
              */
             required: {
               type: Boolean,
+              reflectToAttribute: true,
               value: false
             },
 
             /**
              * The error message to display when the input is invalid.
              */
-            errorMessage: String,
+            errorMessage: {
+              type: String,
+              reflectToAttribute: true
+            },
```

Comment:
Why would this be reflected to attribute? Does any other component do it? At least not text field or date picker.

Reply:
This was done for testing purposes and pushed to remote to save progress (PR is still `in progress`). Removed now.